packages/rate-limit-controller/src/RateLimitController.ts

```diff
@@ -132,16 +142,18 @@ export class RateLimitController<
   async call<ApiType extends keyof RateLimitedApis>(
     origin: string,
     type: ApiType,
-    ...args: Parameters<RateLimitedApis[ApiType]>
-  ): Promise<ReturnType<RateLimitedApis[ApiType]>> {
+    ...args: Parameters<RateLimitedApis[ApiType]['method']>
+  ): Promise<ReturnType<RateLimitedApis[ApiType]['method']>> {
     if (this.isRateLimited(type, origin)) {
       throw ethErrors.rpc.limitExceeded({
-        message: `"${type}" is currently rate-limited. Please try again later.`,
+        message: `"${
+          type as string
```

Comment:
How come this cast is necessary now?

Reply:
I don't know, it's now considered as `symbol`

Reply:
If it's a symbol, we use `.toString()` instead of casting it?

Reply:
> If it's a symbol, we use .toString() instead of casting it?

Yes:

```ts
const foo = Symbol('foo');
const bar = `${foo}`; // TypeError: Cannot convert a Symbol value to a string
```

Reply:
Ugh, typos. *Can* we use.